src/test/java/com/karankumar/booksapi/repository/BookRepositoryTest.java

```diff
@@ -74,13 +72,31 @@ void findSavedBooks() {
             softly.assertThat(result).containsExactlyInAnyOrder(book);
         });
     }
+
+    @Test
+    @DisplayName("find book with isbn")
+    void findBookByIsbn() {
+        createAndSaveAuthors();
+        Book book = createBookwithISBN();
+        bookRepository.save(book);
+
+        Book result = bookRepository.findBookByISBN("978-3-16-148410-0");
+        Assertions.assertEquals(book,result);
+
+    }
   
     private void createAndSaveAuthors() {
         author1 = new Author("Kevlin", "Henney");
         author2 = new Author("Trisha", "Gee");
         saveAllAuthors(author1, author2);
     }
 
+    private Book createBookwithISBN() {
+        Book book = new Book("Game of APIs", new Author[]{author1, author2} , Language.ENGLISH);
+        book.setIsbn13("978-3-16-148410-0");
```

Comment:
Instead of having the ISBN literal twice, could you please extract this to a field?